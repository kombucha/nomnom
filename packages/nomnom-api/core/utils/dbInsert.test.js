const insert = require("./dbInsert");

it("should respect the number of values", () => {
  const [_, values] = insert("TableName", { something: true, else: 2 });
  expect(values.length).toBe(2);
});

it("should not change output", () => {
  const result = insert("TableName", { something: true });
  expect(result).toMatchSnapshot();
});
