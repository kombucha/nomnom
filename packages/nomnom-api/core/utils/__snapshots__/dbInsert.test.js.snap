// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not change output 1`] = `
Array [
  "
    INSERT INTO \\"TableName\\"(\\"something\\")
    VALUES ($1)
  ",
  Array [
    true,
  ],
]
`;
